services/QuillLMS/app/serializers/progress_reports/standards/student_serializer.rb

```diff
@@ -18,7 +18,7 @@ class ProgressReports::Standards::StudentSerializer < ActiveModel::Serializer
              :mastery_status
 
   def average_score
-    object.average_score.round(2)
+    object.average_score&.round(2) || 0
```

Comment:
Are we sure we want to default to 0 for a "score" value?  There's definitely a difference between "you have no score" and "you scored 0%" in some contexts.  Is this one of them?

Reply:
Good question - I'm currently working on a related ticket that will change display logic: https://www.notion.so/quill/Product-Board-30f97e4eb01246dbb8264253fb385073?p=2d5f2d4bf8874dfd8d854fdb5ef546b7